flowetl/etl/etl/config_parser.py

```diff
@@ -36,12 +36,6 @@ def validate_config(*, global_config_dict: dict) -> Exception:
             ValueError("default_args must be a toplevel key in the config file")
         )
 
-    etl_keys = global_config_dict.get("etl", {}).keys()
-    if etl_keys != CDRType._value2member_map_.keys():
```

Comment:
Just to check, are the keys still a subset of the CDR_TYPE possible? Would suggest checking for something like:
```set(etl_keys).issubset(set(CDRType._value2member_map_.keys()))```

Reply:
Yes, that's a great point. Indeed, they are still just a subset. This check is implicitly done elsewhere in the code (where we convert them via `CDRType(cdr_type)`, which will fail if that condition is not met) but good to have an explicit check here, too. 👍